tools/obisindicators/obisindicators.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="obisindicators" name="Ocean biodiversity indicators" version="@VERSION@" profile="20.01">
+    <description>from OBIS</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="topic"/>
+    <expand macro="requirements">
+        <requirement type="package" version="2.0.3">r-magrittr</requirement>
+        <requirement type="package" version="1.0.10">r-dplyr</requirement>
+        <requirement type="package" version="1.0_7">r-sf</requirement>
+        <requirement type="package" version="2.1_7.1">r-gsl</requirement>
+        <requirement type="package" version="0.1.0">r-rnaturalearth</requirement>
+        <requirement type="package" version="0.1.0">r-rnaturalearthdata</requirement>
+        <requirement type="package" version="0.6.2">r-viridis</requirement>
+        <requirement type="package" version="3.0.0">r-dggridr</requirement>
+    </expand>
+    <required_files>
+        <include type="literal" path="test-data/occ_at.tabular"/>
```

Comment:
Only files that are really necessary for the tool to work. Test data files are not required to execute a tool isn't it?

Reply:
absolutely not ! I'll remove them